packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -133,78 +140,93 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     }
   }
 
+  public componentDidMount(): void {
+    if (this._isChartEmpty) {
+      d3Select('body')
+        .append('div')
+        .attr('role', 'alert')
+        .attr('id', 'ariaLabel_AreaChart')
+        .style('opacity', 0)
+        .attr('aria-label', 'Graph has no data to display')
+        .attr('tabIndex', 0);
```

Comment:
Can the functionality be achieved without using tab index

Reply:
removed tabIndex throughout all charts